### PR TITLE
Implement t.hover (part of #240)

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @inikulin @AlexanderMoskovkin

### CODE REVIEW

File: test/functional/fixtures/api/es-next/hover/test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var expect = require('chai').expect;
+
+describe('[API] Hover action', function () {
+    it('Should run hover over containers', function () {
```
Comment (georgiy-abbasov) [inline test/functional/fixtures/api/es-next/hover/test.js]: This test should run only in chrome too.

**Comment by inikulin** [comment]
Oh, just found out that we need to specify chrome only for doubleClick as well. @AndreyBelym can you do this on context of this PR as well?

**Comment by inikulin** [comment]
And rightClick...

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Comment by inikulin** [comment]
**Update:** nevermind. I thought we have #445 merged

File: test/functional/fixtures/api/es-next/hover/test.js (hunk 1 above)
Comment (inikulin) [inline test/functional/fixtures/api/es-next/hover/test.js]: @AndreyBelym please, address this problem

Comment (AndreyBelym) [inline test/functional/fixtures/api/es-next/hover/test.js]: OK, I'll add `[ONLY]` tag here, and also in rClick/dblClick.

**Comment by AndreyBelym** [comment]
lgtm

**Comment by AndreyBelym** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm
/cc @VasilyStrelyaev

**Comment by inikulin** [comment]
lgtm

**Comment by inikulin** [comment]
Require change to functional tests due to #445

**Comment by AlexanderMoskovkin** [comment]
#445 is merged. Please change `[ONLY:chrome]` directives in mocha and testcafe test names with { only: 'chrome' } argument in the `runTests` function

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by AndreyBelym** [comment]
FPR

File: test/functional/fixtures/api/es-next/hover/test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        return runTests('./testcafe-fixtures/hover-test.js', 'Hover over containers', { shouldFail: true, only: 'chrome' })
+            .catch(function (errs) {
+                expect(errs[0]).to.contains(' >  8 |    await t.hover(\'#container2\');');
```
Comment (AlexanderMoskovkin) [inline test/functional/fixtures/api/es-next/hover/test.js]: @AndreyBelym Should we check here the error message (`Hover on inputs raised`)? The test is ok now if any other error is raised? (for example `Hover selector is not applied to #container2`)

Comment (AndreyBelym) [inline test/functional/fixtures/api/es-next/hover/test.js]: Strange, looks like I deleted accidentally a such line. Of course, it should be here.

**Comment by AlexanderMoskovkin** [comment]
/r- (just on missed check in the test)

**Comment by AlexanderMoskovkin** [comment]
tests failed because of unstable OS X 10.10

**Comment by AlexanderMoskovkin** [comment]
lgtm

File: test/functional/fixtures/api/es-next/hover/testcafe-fixtures/hover-test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// NOTE: to preserve callsites, add new tests AFTER the existing ones
+
+fixture `Double Click`
```
Comment (inikulin) [inline test/functional/fixtures/api/es-next/hover/testcafe-fixtures/hover-test.js]: `Hover`

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**